packages/amplify-provider-awscloudformation/src/amplify-service-manager.ts

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable jsdoc/require-jsdoc */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import {
+  $TSContext, AmplifyError, amplifyFaultWithTroubleshootingLink, pathManager, stateManager,
+} from 'amplify-cli-core';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { printer, prompter } from 'amplify-prompts';
+import { ensureEnvMeta } from '@aws-amplify/amplify-environment-parameters';
```

Comment:
This file has been converted from js to ts (apologies for not doing that in a separate PR)
The only functional change is loading amplfiy meta from `ensureEnvMeta` instead of directly from the file using stateManager. This happens in `deleteEnv` and `postPushCheck`